### PR TITLE
Update changelog for 4.35.0

### PR DESCRIPTION
**Pull request by benjaminulmer**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,35 @@
...
+- Windows support for benchmarking client
+- DirectToVgpr support for CGEMM
+- TensileLibLogicToYaml for creating tuning configs from library logic solutions
+### Optimizations
+- Improved performance for StoreCInUnroll + b128 store
+### Changed
+- No syncThreads if number of threads in block equal to wavefront size
+- Re-enable HardwareMonitor for gfx90a
+- Decision trees use MLFeatures instead of Properties
+- Put beta code and store separately if StoreCInUnroll = x4 store
```
Comment (nakajee) [inline CHANGELOG.md:23]: This is intended to optimize the performance.
Probably, better to put under optimizations.

Comment (benjaminulmer) [reply CHANGELOG.md]: Moved lines 20 and 23 to Optimizations

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,35 @@
...
+- Support for CGEMM + MIArchVgpr
+- client-path parameter for using prebuilt client
+- CleanUpBuildFiles global parameter
+- Debug flag for printing library logic index of winning solution
+- NumWarmups global parameter for benchmarking
+- Windows support for benchmarking client
+- DirectToVgpr support for CGEMM
+- TensileLibLogicToYaml for creating tuning configs from library logic solutions
+### Optimizations
+- No syncThreads if number of threads in block equal to wavefront size
```
Comment (nakajee) [inline CHANGELOG.md:18]: I think this is reverted.

Comment (nakajee) [reply CHANGELOG.md:18]: > I think this is reverted.

Precisely, commented out.

Comment (nakajee) [reply CHANGELOG.md]: Thanks for your update.
I do not have any further comment.
Waiting for others.

**Pull request merged**